contrib/face_recognition.lua

```diff
@@ -439,18 +450,13 @@ table.insert(widgets, dt.new_widget("box"){
 table.insert(widgets, fc.execute)
 
 fc.widget = dt.new_widget("box"){
-  orientation = vertical,
-  reset_callback = function(this)
+	orientation = vertical,
+	reset_callback = function(this)
     reset_preferences()
-  end,
-  table.unpack(widgets),
+	end,
+	table.unpack(widgets),
```

Comment:
same as above

Reply:
I used an online beautifier to make sure all my changes were correctly indented. It also changed some of your code below “-- build the interface”. I will rollback it.